HashVerify/Verify-Hash.psm1

```diff
@@ -0,0 +1,227 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+
+<#
+This module contains the function to verify expected hash of the file specified in the provided path.
+#>
+
+
+function Verify-Hash {
```

Comment:
Test-FileHash

Reply:
will change it as Test is an approved verb for cmdlet. Thank you :)

Reply:
Make sure you also update the readme example as you change the verb to test

Reply:
have updated